hyperspy/_signals/signal2d.py

```diff
@@ -605,3 +608,433 @@ def add_ramp(self, ramp_x, ramp_y, offset=0):
         ramp += ramp_x * xx
         ramp += ramp_y * yy
         self.data += ramp
+
+    def find_peaks2D(self, method='skimage', *args, **kwargs):
+        """Find peaks in a 2D signal/image.
+        Function to locate the positive peaks in an image using various, user
+        specified, methods. Returns a structured array containing the peak
+        positions.
+        Parameters
+        ---------
+        method : str
+                 Select peak finding algorithm to implement. Available methods
+                 are:
+                     'max' - simple local maximum search
+                     'skimage' - call the peak finder implemented in
+                                 scikit-image which uses a maximum filter
+                     'minmax' - finds peaks by comparing maximum filter results
+                                with minimum filter, calculates centers of mass
+                     'zaefferer' - based on gradient thresholding and refinement
+                                   by local region of interest optimisation
+                     'stat' - statistical approach requiring no free params.
+                     'massiel' - finds peaks in each direction and compares the
+                                 positions where these coincide.
+                     'laplacian_of_gaussians' - a blob finder implemented in
+                                                `scikit-image` which uses the
+                                                laplacian of Gaussian matrices
+                                                approach.
+                     'difference_of_gaussians' - a blob finder implemented in
+                                                 `scikit-image` which uses
+                                                 the difference of Gaussian
+                                                 matrices approach.
+        *args : associated with above methods
+        **kwargs : associated with above methods.
+        Returns
+        -------
+        peaks: structured array of shape _navigation_shape_in_array in which
+               each cell contains an array with dimensions (npeaks, 2) that
+               contains the x, y coordinates of peaks found in each image.
```

Comment:
Are the coordinates in indices? Looking at the methods I assume so, however from the description I would guess physical / calibrated coordinates.

Also, subpixel resolution or not?